### PR TITLE
Fix JaCoCo configuration default values sample

### PR DESCRIPTION
**Pull request by britter**
Before this only the Kotlin DSL sample was rendered without syntax
highlighting. This change moves the samples to the sample project in
order to get rendering for both DSL variants.

This is how it looks at the moment:

Bildschirmfoto 2019-10-15 um 15 54 13

...and this is how it looks after this change:

Bildschirmfoto 2019-10-15 um 16 04 30

### CODE REVIEW

File: subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts [hunk 1]
```diff
@@ -65,3 +65,23 @@ tasks.jacocoTestCoverageVerification {
     }
 }
 // end::violation-rules-configuration[]
+
+// tag::testtask-configuration-defaults[]
+tasks.getByName<Test>("test") {
```
Comment (eskatos) [inline subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts]: :x: should use the accessor to the `test` task
```suggestion
tasks.test {
```

File: subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts [hunk 2]
```diff
@@ -65,3 +65,23 @@ tasks.jacocoTestCoverageVerification {
...
+    extensions.configure(JacocoTaskExtension::class) {
```
Comment (eskatos) [inline subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts]: :x: should use `configure<T> {}`, there's another instance of that above line 27
```suggestion
    configure<JacocoTaskExtension> {
```

File: subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts [hunk 3]
```diff
@@ -65,3 +65,23 @@ tasks.jacocoTestCoverageVerification {
...
+        isEnabled = true
+        destinationFile = file("$buildDir/jacoco/$name.exec")
+        includes = listOf()
```
Comment (eskatos) [inline subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts]: :o: using `emptyList()` might be clearer, applies to other list properties below
```suggestion
        includes = emptyList()
```

**Comment by britter** [comment]
Waiting for a green CI build on master...

**Comment by britter** [comment]
@eskatos PTAL and merge if your satisfied 🙂

**Pull request merged**